components/common/pkg/kafka/health/health_test.go

```diff
@@ -0,0 +1,198 @@
+package health
+
+import (
+	"fmt"
+	"github.com/kyma-incubator/knative-kafka/components/common/pkg/log"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+)
+
+// Test Constants
+const (
+	testHttpPort    = "8089"
+	testHttpHost    = "localhost"
+	livenessPath    = "/healthz"
+	readinessPath   = "/healthy"
+)
```

Comment:
are these endpoint names standard?  they're ok - liveness seems a little odd but...

Reply:
I don't know if they're "standard" but they are from https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
these constants are re-declared in constants file down below (in the controller and i presume the dispatcher) can we use the ones in common instead?  Do we expect/need/want different values for them in either place?